Merge party scale extension, update organization classifications example (#1459)
open-contracting/standard: docs/guidance/map/organization_classifications.md

Many publishers wish to disclose and monitor information about different classifications of organizations involved in contracting processes. Example classifications include organization ownership, incorporation, sector, location, and number of employees. We recognise that publishing information about a variety of organization characteristics, for example women-owned and/or indigenous-owned organizations, is an important part of monitoring participation in public procurement.

Some organization classifications, such as organization scale, can be published using OCDS fields and codes. Many other classifications are context-specific, and for this case, we provide an organization classification extension that allows publishers to provide their own classifications.

There are therefore two options that are encouraged for publishing organization classifications.

> 1. For classifications that have become standardized, there are specific OCDS fields and codes that ought to be used. At present, this only applies to organization scale, which ought to be disclosed in `parties.details.scale`, using a code from the [party scale codelist](../../schema/codelists.md#party-scale).

Reviewer: ```suggestion
1. For classifications that have become standardized, there are specific OCDS fields and codes that ought to be used. At present, this only applies to organization scale, which ought to be disclosed using the `parties.details.scale` field, using a code from the [party scale codelist](../../schema/codelists.md#party-scale).
```

#### Organization scale

> In the example below, Moldova has disclosed information about the 'Companie mică' organization using `parties.details.scale`. The scale is given as 'micro', from the [partyScale codelist](../../schema/codelists.md#party-scale).

Reviewer: ```suggestion
In the example below, Moldova has disclosed information about the 'Companie mică' organization using the `parties.details.scale` field, to which it has assigned the 'micro' code from the [partyScale codelist](../../schema/codelists.md#party-scale).
```
